packages/ui/src/services/config-transport/createProxyComponent.ts

```diff
@@ -0,0 +1,66 @@
+import { getCurrentInstance, ComponentInternalInstance, DefineComponent, SetupContext, Ref } from 'vue'
+import { useComponentConfigProps } from '../component-config/component-config'
+import { computedShallowReactive } from './computedShallowReactive'
+
+/** Compiled and reactive props. By default they passed to setup fn */
+type Props = Record<string, unknown>;
```

Comment:
Is this type somehow related to the type `Props` from the file `../component-config/component-config.ts`?

Reply:
This type actually must be:
```suggestion
type Props = Parameters<NonNullable<DefineComponent<Record<string, unknown>>['setup']>>[0];
```

But it's a very redundant 😅.